sqlx-cli/src/database.rs

```diff
@@ -30,3 +31,13 @@ pub async fn drop(uri: &str, confirm: bool) -> anyhow::Result<()> {
 
     Ok(())
 }
+
+pub async fn reset(uri: &str, confirm: bool) -> anyhow::Result<()> {
+    drop(uri, confirm).await?;
+    setup(uri).await
+}
+
+pub async fn setup(uri: &str) -> anyhow::Result<()> {
+    create(uri).await?;
```

Comment:
Do you think we should modify `create` to add `if not exists` for this call? We could later add that as a flag to `sqlx db create` at well.

Reply:
Ah good catch, definitely! In Phoenix it will log a message that it is already created, and continue running any migrations. I'll get a change pushed tomorrow morning probably.

Reply:
Sorry it was late last night so I didn't verify, but I just double checked and I believe `create` is already doing the `if not exists` check. I don't think anything needs to be changed for what we want aside from adding that flag eventually.

Reply:
Ah, that's fine then.